Cannot read property 'length' of undefined customHandlebarsExtensionFile

**Describe the bug**
When trying to generate release notes (crossplatform), the reference to the customHandlebarsExtensionFile from the newest PR tries to determine the length of a null string and fails to complete.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to Azure Pipelines
2. Visit a Generate Release Notes (Crossplatform) task
3. Use task version 3.*
4. Template Location = InLine
5. Include a template with references to releaseDetails object
6. Under "Handlebars", enter no references to any custom handlebars extensions, files, folders, etc
7. Run the task and see it fail on ReleaseNotesFunctions.ts line 769

**Expected behavior**
The code should do a null check on the newly implemented custom handlebar functionality so that the task can succeed without requiring the custom handlebar fields to be filled in

**Logged Error**
2021-03-01T23:37:35.8340128Z Loading handlebars-helpers extension
2021-03-01T23:37:36.1661161Z ##[error]TypeError: Cannot read property 'length' of undefined

**Logs**
See attached file error.log
[error.log](https://github.com/rfennell/AzurePipelines/files/6069646/error.log)

**What is the issue see:**
 - Azure DevOps Services (VSTS)
 - Version 3.44.4

--- a/Extensions/XplatGenerateReleaseNotes/V3/ReleaseNotesFunctions.ts
+++ b/Extensions/XplatGenerateReleaseNotes/V3/ReleaseNotesFunctions.ts
@@ -766,12 +766,12 @@ export function processTemplate(
         );
 
         // make sure we have valid file name for the custom extension
-        if (customHandlebarsExtensionFile.length > 0) {
+        if (! customHandlebarsExtensionFile || customHandlebarsExtensionFile.length === 0) {
+            customHandlebarsExtensionFile = "customHandlebarsExtension.js";
+        } else {
             if (!customHandlebarsExtensionFile.toLowerCase().endsWith(".js")) {
                 customHandlebarsExtensionFile = customHandlebarsExtensionFile + ".js";
             }
-        } else {
-            customHandlebarsExtensionFile = `${customHandlebarsExtensionFile}.js`;
         }
 
         if (!customHandlebarsExtensionFolder || customHandlebarsExtensionFolder.length === 0) {

